birdhouse/components/finch/service-config.json.template

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
   "name": "finch",
+  "version": "${FINCH_VERSION}",
+  "types": [
+    "wps"
+  ],
   "keywords": [
```

Comment:
should we leave these keywords as is if we're moving this information to "types". What if we borrowed some of the tags and categories from canarie-api instead. So for finch we could do something like:

```
"keywords": ["Climatology", "Cloud", "Data Slicer", "Climate Indices", ...]
```

Reply:
Yes. I would like to do this, but until there is a clear-cut version I can refer to, adding these to the keywords will fail schema validation.

Reply:
Got it. Try now with version [1.2.0](https://github.com/DACCS-Climate/Marble-node-registry/releases/tag/1.2.0)?

Reply:
Are we going to do this in this PR or in a different one?

Reply:
If possible, I'd prefer to work on this only once.